### ISSUE TITLE
[decl'HDF5] HDF5 compression

### ISSUE THREAD

**Issue by jbigot** [pdidev/pdi#449]
In GitLab by @mtrochon on Jun 17, 2024, 10:34

I am trying to use compression in Gysela. I modified the yaml file as the image shows, in the output log it's written that Decl'HDF5 does use deflate. But I don't see any changes regarding the time and the output file sizes. 
[image](https://gitlab.maisondelasimulation.fr/pdidev/pdi/uploads/554c7f6f31bc37c264ba436987192b3e/image.png)

**Comment by jbigot** [comment]
Can you provide us with your full yaml & log in text format, please?

**Comment by jbigot** [comment]
also, can you actually check the compression level of each datacheck as described in https://stackoverflow.com/questions/70202779/determine-if-a-hdf5-file-created-with-h5py-is-compressed-or-not

**Comment by jbigot** [comment]
In GitLab by @mtrochon on Jul 2, 2024, 09:28

Good morning,

I checked the compression and indeed it works.
The problem was that i compile gysela+GNU+PDI which gave a very bad compression (from 1.1T to 1.05T)
So, I compare it with gysela+Intel+HDF5, where i got good results (from 1.1T to 500GB)
Then, i checked gysela+GNU+HDF5, and I got the same result as the first tests.

So the problem is between GNU and HDF5_compression, and I will look more into it. Sorry for the trouble !
